src/app/iotHub/sagas/getSharedAccessSignatureAuthorizationRulesSaga.ts

```diff
@@ -0,0 +1,58 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import { call, select, put } from 'redux-saga/effects';
+import { setSharedAccessSignatureAuthorizationRulesAction } from '../actions';
+import { getSharedAccessSignatureAuthorizationRules } from '../services/iotHubService';
+import { AzureResourceIdentifier } from '../../azureResourceIdentifier/models/azureResourceIdentifier';
+import { SharedAccessSignatureAuthorizationRule } from '../models/sharedAccessSignatureAuthorizationRule';
+import { executeAzureResourceManagementTokenRequest } from '../../login/services/authService';
+import { appConfig } from '../../../appConfig/appConfig';
+import { StateInterface } from '../../shared/redux/state';
+import { CacheWrapper } from '../../api/models/cacheWrapper';
+
+const cacheInMinutes = 4;
+const secondsPerMinute = 60;
```

Comment:
60 and 1000 are two constants in utils.ts which can be exported and used here.

Reply:
np -- going to move these to constants as exported values (currently not exported);